content/docs/getting-started/installing.md

```diff
@@ -5,64 +5,85 @@ weight: 2
 
 ## From extension stores
 
-Scratch Addons is available in both Chrome Web Store and Add-ons for Firefox.
+Scratch Addons is available in both Chrome Web Store and Firefox Add-ons.
 
-- Chrome Web Store (for Google Chrome, Opera, Brave, Vivaldi, Microsoft Edge (>79), and other Chromium-based browsers)  
+- Chrome Web Store (for Google Chrome, Opera, Brave, Vivaldi, Microsoft Edge Chromium (>79), and other Chromium-based browsers)  
```

Comment:
\>79 already implies Chromium Edge. Also, the fact that I put it on a list with a mention of Chrome/ium already supports it.

Reply:
Fixed, will commit soon.